airmozilla/main/templates/main/event.html

```diff
@@ -334,6 +335,24 @@ <h1 class="entry-title">
         </div>
       {% endif %}
 
+      {% if event.is_public() %}
+        <div class="tab">
+          <input type="radio" id="tab-share" name="tab-group-1">
+          <label for="tab-share">Share</label>
+          <div class="content">
+            <div>
+              <a class="share" href="https://www.facebook.com/sharer/sharer.php?u={{ make_absolute(url('main:event', event.slug)) }}" target="_blank">Share on Facebook</a>
```

Comment:
We're not using that built-in and the `url()` function will return something like '/foo' which is fine. The the `make_absolute_url('/foo')` will return `http://localhost:8000/foo` which is also fine but that means that the final HTML becomes `href="https://www.facebook.com/sharer/sharer.php?u=http://localhost:8000/foo"` when instead it should be `href="https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Flocalhost%3A8000%2Ffoo"`